### DISCUSSION TITLE
Const declaration support

### DISCUSSION THREAD

**Discussion by killkrt** [gfx-rs/naga#2039] [General]
Hi,

I've notice that [const](https://gpuweb.github.io/gpuweb/wgsl/#const-declaration) is not yet supported by Naga, it is by design or it should be tracked by a specific issue?

Thank you

**Comment by teoxoy** [comment]
Hey!

It's tracked by #1829 since const expressions and declarations go hand in hand. Currently you can still use let in global scope but only with literals.
